pyxis_slurmstepd.c

```diff
@@ -561,6 +554,12 @@ static pid_t enroot_exec(uid_t uid, gid_t gid, char *const argv[])
 
 	enroot_reset_log();
 
+	char *argv_str = join_strings(argv, " ");
```

Comment:
So far we don't mix declarations and code, so let's keep it this way (I know, we're not using C90).